changes-test-results.diff

```diff
@@ -0,0 +1,42 @@
+diff --git a/tests/kinematically_driven_subduction_2d_case1/screen-output b/tests/kinematically_driven_subduction_2d_case1/screen-output
```

Comment:
This file should not be added to the git history. Can you remove it by running `git rm ...` and then amend the commit (`git commit --amend`) and force push to this PR?